curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/check-for-the-presence-of-an-element-with-indexof.english.md

```diff
@@ -31,6 +31,8 @@ fruits.indexOf('pears'); // returns 1, the first index at which the element exis
 
 ```yml
 tests:
+  - text: The <code>quickCheck</code> function should return a boolean (<code>true</code> or <code>false</code>), not a string (<code>"true"</code> or <code>"false"</code>)
+    testString: assert.strictEqual(quickCheck(['squash', 'onions', 'shallots'], 'mushrooms'), false);
```

Comment:
With this text, I think the test needs to just confirm that the output is `boolean`.  If a user makes a mistake and has `quickCheck(['squash', 'onions', 'shallots'], 'mushrooms') === true`, then it would be confusing to be told they "should return a boolean" when they *have*, just the wrong one.

Reply:
That's a really good point that I hadn't considered - I'll re-evaluate and adjust accordingly. :)